### PR TITLE
#192 : Remove claim  command in Daemon 

### PR DESCRIPTION
**Pull request by anandrgitnirman**
With all claims now managed by snet-cli , Obsolete Code to claim  in Daemon is being removed 
https://github.com/singnet/snet-daemon/issues/192

### CODE REVIEW

File: metrics/heartbeat.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+
```
Comment (raamb) [inline metrics/heartbeat.py]: Why is the heartbeat file appearing in the claim PR

**Comment by vsbogd** [comment]
Looks good to me, except `heartbeat.py` file

File: heartbeat.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+
```
Comment (vsbogd) [inline heartbeat.py]: It seems you moved it instead of removing

Comment (anandrgitnirman) [reply heartbeat.py]: my bad :) , removed it now for sure , also removed references to private keys and mnemonic

**Pull request merged**